src/interface/gemm_interface.hpp

```diff
@@ -115,22 +119,48 @@ typename sb_handle_t::event_t _gemm_backend(
 
   bool _TrA = _TransA != 'n';
   bool _TrB = _TransB != 'n';
+
+  // Checking Strides conformity when batch_size>1
+  index_t a_size = _TrA ? _M * _lda : _K * _lda;
+  index_t b_size = _TrB ? _ldb * _K : _N * _ldb;
+  index_t c_size = _ldc * _N;
+
+  if (batch_size > index_t(1)) {
+    if (batch_type == gemm_batch_type_t::strided) {
+      if (_stridec < c_size || _stridec < 0) {
+        throw std::invalid_argument("invalid _stridec");
+      } else if (_stridea < 0) {
```

Comment:
I think we need to make the if conditions for `stride_a` and `stride_b` a bit more exhaustive. The user can set a stride which is much larger than the `batch size` or the `lda` and I am not sure that the GEMM implementation would be able to cover those cases.

Reply:
Following the most recent oneMKL's `gemm_batch` [spec](https://spec.oneapi.io/versions/latest/elements/oneMKL/source/domains/blas/gemm_batch.html#onemkl-blas-gemm-batch), there are no conditions on the strides values, as the user is responsible of passing a valid buffer that can be accessed given the batch_size, matrices sizes and the stride values.